smac/intensification/abstract_racer.py

```diff
@@ -117,43 +117,75 @@ def __init__(self,
         # to mark the end of an iteration
         self.iteration_done = False
 
-    def eval_challenger(self,
-                        challenger: Configuration,
-                        incumbent: typing.Optional[Configuration],
-                        run_history: RunHistory,
-                        time_bound: float = float(MAXINT),
-                        log_traj: bool = True) -> typing.Tuple[Configuration, float]:
-        """
-        Runs intensification by evaluating one configuration-instance at a time
+    def eval_challenger(
```

Comment:
Yes, I will move it. I left it here because I assume we eventually will use a worker scheme for it and delete it, but let me move it to the main loop.